modules/ROOT/pages/7.1.2-release-notes.adoc

```diff
@@ -55,17 +55,21 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
 
 The following premium plugin updates were released alongside {productname} {release-version}.
 
-=== <Premium plugin name 1> <Premium plugin name 1 version>
+=== Spell Checker
+// TINY-10904
 
-The {productname} {release-version} release includes an accompanying release of the **<Premium plugin name 1>** premium plugin.
+The {productname} {release-version} release includes an accompanying release of the **Spell Checker** premium plugin.
 
-**<Premium plugin name 1>** <Premium plugin name 1 version> includes the following <fixes, changes, improvements>.
+**Spell Checker** includes the following fix.
 
-==== <Premium plugin name 1 change 1>
+==== Added test for acronym handling in `tinymcespellchecker`
```

Comment:
We don't need to document when we add or modify tests

Reply:
Will remove this section.